### DISCUSSION TITLE
Innr RS 128 T exposes color_xy even though the bulb only supports color temp

### DISCUSSION THREAD

**Discussion by johannes-mueller** [Koenkk/zigbee2mqtt#22205] [Devices]
Hi,

I am in the process of migrating from deconz to Z2M since literally today. So please don't hesitate to just point me to docs or FAQs if I seem to have missed them.

I have a couple of [Innr RS 128 T](https://www.zigbee2mqtt.io/devices/RS_128_T.html) in my apartment which are found by Z2M promptly. However, according to Z2M the RS 128 T exposes the `color_xy` parameter even though the device actually only supports `color_temp`. Why is that? Are there models with the identical model name that do support real color? Can I change that locally because the appearance of the light in the HA frontend is misleading.

Edit:

I found a hacker's workaround: I deleted the ` color: { applyRedFix: true }` entry in entry of the RS 128 T in `node_modules/zigbee-herdsman-converters/devices/innr.js`, which makes the light appear correctly in the HA frontend, but that does not seem to be a sustainable solution.

**Comment by Koenkk** [comment]
Done!

Changes will be available in the [dev branch](https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html) in a few hours from now.

**Reply by johannes-mueller** [reply]
Thanks for the quick resolution.
